src/resources/app/pages/course/EditCourseMaterial.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <CreationLayout
+    :courseId="this.$props.id"
+    pageTitle="Update Material"
+  >
+    <v-form ref="form" v-model="isValid">
+      <v-autocomplete
+        id="drop-down"
+        v-model="form.topic_id"
+        :items="items"
+        label="Topic"
+        outlined
+        :item-text="getItemText"
+        :item-value="getItemValue"
+        required
```

Comment:
nit, please change these to:
```
        id="drop-down"
        v-model="form.topic_id"
        label="Topic"
        outlined
        required
        :items="items"
        :item-text="getItemText"
        :item-value="getItemValue"
```

put all attributes that use dynamic elements together